libs/MotionKit/include/MotionKit.h

```diff
@@ -47,9 +52,9 @@ class MotionKit
 	bool _rotate_x_turns_requested = false;
 
 	const float kReferenceAngle		   = 180.F;
-	const float kMinimalViableRobotPwm = 0.25F;	  // ? Under this pwm value, torque is too low to spin the wheel
+	const float kMinimalViableRobotPwm = 0.15F;	  // ? Under this pwm value, torque is too low to spin the wheel
 	const float kPwmMaxValue		   = 1.F;
-	const float kPwmMarginLimit		   = 0.28F;
+	const float kPwmMarginLimit		   = 0.17F;
```

Comment:
why those values? what changed?

Reply:
Yes, why this is more performant than last time? Because the robot is closed?